core/ops.rs

```diff
@@ -158,6 +159,7 @@ pub struct OpState {
   pub resource_table: ResourceTable,
   pub get_error_class_fn: GetErrorClassFn,
   pub tracker: OpsTracker,
+  pub last_fast_op_error: Option<AnyError>,
```

Comment:
Heads up, there is a custom field introduced in https://github.com/denoland/deno/pull/15976 that does the same. If we go with your solution that fields should be removed (should that TTY PR land first)

Reply:
the TTY PR is essentially doing the same thing as this PR but manually (not inside the proc macro). We should land this PR first.